File: app/src/main/java/com/theandroiddev/mywins/images/ImageActivity.java
package com.theandroiddev.mywins.images;

import android.os.Bundle;
import android.support.v4.view.ViewPager;
import android.support.v7.app.AppCompatActivity;

import com.theandroiddev.mywins.R;

import java.util.ArrayList;

public class ImageActivity extends AppCompatActivity {
    private static final String TAG = "ImageActivity";

    ViewPager viewPager;
    ImageSwipeAdapter imageSwipeAdapter;
    ArrayList<String> imagePaths;
    int pos;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_image);

        imagePaths = getIntent().getStringArrayListExtra("imagePaths");
        pos = getIntent().getIntExtra("position", 0);
        viewPager = findViewById(R.id.view_pager);
        imageSwipeAdapter = new ImageSwipeAdapter(this, imagePaths);
        viewPager.setAdapter(imageSwipeAdapter);
        viewPager.setCurrentItem(pos);
        viewPager.setOffscreenPageLimit(2);

    }

}

File: app/src/main/java/com/theandroiddev/mywins/presentation/image/ImageActivity.kt
package com.theandroiddev.mywins.presentation.image

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.theandroiddev.mywins.R
import kotlinx.android.synthetic.main.activity_image.*

class ImageActivity : AppCompatActivity() {

    private var imageSwipeAdapter: ImageSwipeAdapter? = null
    private var pos: Int = 0

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_image)

        val imagePaths = intent.getStringArrayListExtra("imagePaths")
        pos = intent.getIntExtra("position", 0)
        imageSwipeAdapter = ImageSwipeAdapter(this, imagePaths)
        view_pager.adapter = imageSwipeAdapter
        view_pager.currentItem = pos
        view_pager.offscreenPageLimit = 2

    }

}
